context_test.go

```diff
@@ -374,6 +378,11 @@ func TestDetectProjectGOPATH(t *testing.T) {
 
 	h.TempDir("sym") // Directory for symlinks
 
+	h.TempDir(filepath.Join(".", "sym/symlink"))
+	h.TempDir(filepath.Join("go", "src", "sym", "path"))
+	h.TempDir(filepath.Join("go", "src", " real", "path"))
+	h.TempDir(filepath.Join("go-two", "src", "real", "path"))
+
```

Comment:
Looks like we are repeating directory creation here? Can we remove the old ones?